vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/SpaceManagerTest.java

```diff
@@ -168,13 +170,13 @@ public void test_deleteSpaceById_whenSpaceHasLinks() {
     @Test
     public void test_deleteSpaceById_whenLinksToSpace() {  
         Space space = new Space();
-        spaceLink = new SpaceLink();
         space.setId(spaceId1);
         spaceLink.setId(spaceLinkId1);
         spaceLink.setTargetSpace(space);
         Mockito.when(spaceLinkRepo.getLinkedFromSpaces(spaceId1)).thenReturn(Arrays.asList(spaceLink));
         managerToTest.deleteSpaceById(spaceId1);
         Mockito.verify(spaceLinkRepo).deleteBySourceSpaceId(spaceId1);
+        Mockito.verify(spaceLink).setTargetSpace(null);
```

Comment:
I don't think this will work since space link is not mocked.

Reply:
I set it to mocked and checked.

Reply:
no need to mock it just assert targetspace is null for that object